### PR TITLE
Reduce precision default value

### PR DESCRIPTION
**Pull request by utas-raymondng**

### CODE REVIEW

File: indexer/src/main/java/au/org/aodn/esindexer/utils/GeometryUtils.java [hunk 1]
```diff
@@ -44,7 +44,7 @@ public enum PointOrientation {
 
     @Getter
     @Setter
-    protected static double coastalPrecision = 0.05;
+    protected static double coastalPrecision = 0.1;
```
Comment (vietnguyengit) [inline indexer/src/main/java/au/org/aodn/esindexer/utils/GeometryUtils.java:47]: should we have a comment to explain why that value or simply a random pick?

**Review by vietnguyengit — APPROVED** [review]

**Comment by vietnguyengit** [comment]
also test failed

File: indexer/src/main/java/au/org/aodn/esindexer/utils/GeometryUtils.java (hunk 1 above)
Comment (utas-raymondng) [reply indexer/src/main/java/au/org/aodn/esindexer/utils/GeometryUtils.java:47]: Added comment

**Comment by utas-raymondng** [comment]
> also test failed

Fixed, miss to set the old default in test to avoid test result different

**Pull request merged**